guides/web-analytics.md

```diff
@@ -0,0 +1,41 @@
+# Web analytics
+
+We use Google Analytics on most of our sites. It's free ([within limits](https://support.google.com/analytics/answer/1070983?hl=en)), it's easy to implement and there aren't many things it can't track.
+
+The login details for access are kept in 1Password, search for "Google Analytics".
+
+## Implementing
+
+We add Google Analytics via [gtag.js](https://developers.google.com/analytics/devguides/collection/gtagjs/). If you're using [whippet-theme](https://github.com/dxw/whippet-theme), the code is already included and you just have to enter the correct tracking ID.
+
+### Event tracking
+
+If there are specific events you want to track that Google Analytics will not track by default (e.g. clicks on links to external sites, file downloads), you will need to implement a custom event, as per [Google's instructions](https://developers.google.com/analytics/devguides/collection/gtagjs/sending-data).
+
+It's important to note the section on [handling timeouts](https://developers.google.com/analytics/devguides/collection/gtagjs/sending-data#handle_timeouts). If you don't do that, the links you are tracking won't work if GA doesn't load (e.g. if the user has a browser extension that blocks trackers).
+
+## Configuring
+
+There are a few basic steps it's worth following for any site you've set up in Google Analytics:
+
+* Set up an unfiltered view that records all traffic, including spam. You won't use this for most analysis, put it provides a control you can compare against, and a backup in case you get the filtering wrong on other views.
```

Comment:
How do I create views?

Reply:
I had to go to `admin` and on the right of the 3 columns select the default 'All Web Site Data' and 'create new view'.